### PR TITLE
Feature/hpapi 308

### PR DESCRIPTION
**Pull request by jeffbum**
remove default values that break dbml output.

instead of `"id" bigint  [default: 'nextval('labreference.base_test_price_id_seq'::regclass)', primary key]`, will know produce `"id" bigint  [primary key]`

### CODE REVIEW

**Review by connormullett — APPROVED** [review]

**Review by connormullett — APPROVED** [review]

**Review by babennettdev — APPROVED** [review]
Jeff, thanks for showing me the end-result of this `pg-to-dbml` branch. The code looks good and I agree with your reasoning that we do not need to show some of the "under the hood" workings of postgres in our DBML.  Approved 👍

File: src/funcs/transformTableStructureToDBML.spec.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      const columnType = { data_type: 'timestamp without time zone'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('timestamp')
+    })
+    it(`returns 'number' when data_type = 'time without time zone'.`, () => {
+      const columnType = { data_type: 'time without time zone'}
+      const response = getColumnType(columnType);
+      expect(response).toEqual('timestamp')
+    })
+    it(`returns undefined if not given coreect object input`, () => {
```
Comment (camoore1979) [inline src/funcs/transformTableStructureToDBML.spec.js]:
```suggestion
    it(`returns undefined if not given correct object input`, () => {
```

File: src/funcs/transformTableStructureToDBML.js [hunk 2]
```diff
@@ -84,11 +80,19 @@ const getColumnDefinition = (col) => {
...
   const columns = colDefs && Array.isArray(colDefs) ? colDefs : [];
   const columnDefinitions = columns.map(column => getColumnDefinition(column, primaryKeys));
   const tableNameString = includeSchemaName ? `${schemaName}.${tableName}` : tableName;
   columnDefinitions.unshift(`Table "${tableNameString}" {`);
   columnDefinitions.push(`} ${EOL} ${EOL} `);
   return columnDefinitions.join(`${EOL} `);
+}
+
+module.exports = {
+  transformTableStructureToDBML,
```
Comment (camoore1979) [inline src/funcs/transformTableStructureToDBML.js:93]: A-Z ordering?

File: src/funcs/transformTableStructureToDBML.js [hunk 3]
```diff
@@ -24,21 +24,17 @@ const getColumnType = (col) => {
...
 }
 
-const cleanUpColumnDefault = columnDefault => columnDefault && columnDefault.includes('::text')
-  ? columnDefault.replace(/::text/gi, '').replace(/'/gi, '')
-  : columnDefault;
 
 const getColumnDefault = (columnDefault, dataType) => {
-  if (!columnDefault) return '';
-  const cleanedUp = cleanUpColumnDefault(columnDefault);
+  if (!columnDefault || columnDefault.includes('::')) return '';
```
Comment (camoore1979) [inline src/funcs/transformTableStructureToDBML.js:29]: So this line here actually fixes the bug described by HPAPI-308, correct?

**Review by camoore1979 — APPROVED** [review]
Approving. It looks good. Thanks for introducing tests!!

A couple minor things, see below.

File: src/funcs/transformTableStructureToDBML.js (hunk 3 above)
Comment (jeffbum) [reply src/funcs/transformTableStructureToDBML.js:29]: well yes and no. Yes, as in, you can now just copy n paste the DBML to db diagram and it should not have any issues (at least with our db, cannot speak broadly 😄 ). The DBML issue was having an extra set of quotes inside a string. This made it seem like some of the strings were keywords DBML should know about. This happened on strings being returned from Postgres that showed what was going on under the hood. All of those strings had the `::` in them. so to kill 2 birds with 1 stone, I decided I saw no need to show end users of the db tool what Postgres does to auto-increment an id.

File: src/funcs/transformTableStructureToDBML.spec.js (hunk 1 above)
Comment (jeffbum) [reply src/funcs/transformTableStructureToDBML.spec.js]: removed tests as they're a part of HPAPI-342.

File: src/funcs/transformTableStructureToDBML.js (hunk 2 above)
Comment (jeffbum) [reply src/funcs/transformTableStructureToDBML.js:93]: In HPAPI-342, these have all been split out to their own files so they will have their own spec files.

**Pull request merged**